### PR TITLE
Mat 4580/add measure cql in test cases

### PR DESCRIPTION
**Pull request by sb-prateekkeerthi**
## MADiE PR

Jira Ticket: [MAT-4580](https://jira.cms.gov/browse/MAT-4580)
(Optional) Related Tickets:

### Summary

Reorganizing the individual test cage(i.e, adding measure cql tab)

### All Submissions
* [x] This PR has the JIRA linked.
* [x] Required tests are included
* [x] No extemporaneous files are included (i.e Complied files or testing results)
* [x] This PR is into the **correct branch**.
* [x] All Documentation as needed for this PR is Complete (or noted in a TODO or other Ticket)
* [ ] Any breaking changes or failing automation are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [ ] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package)
* [ ] All CDN/Web dependencies are hosted internally (i.e MADiE-Root Repo)

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
*  The tests appropriately test the new code, including edge cases
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads

### CODE REVIEW

File: src/components/createTestCase/CreateTestCase.tsx [hunk 1]
```diff
@@ -132,6 +133,17 @@ const INITIAL_VALUES = {
...
 
+interface PropTypes {
+  isActive?: boolean;
+}
+
+const MenuItemContainer = tw.ul`bg-transparent flex px-8`;
+const MenuItem = styled.li((props: PropTypes) => [
+  tw`relative h-2 mr-3 ml-3 mt-3 mb-0 not-italic font-rubik font-medium text-sm leading-4 text-gray`,
+  props.isActive &&
+    tw`bg-white h-2 text-black font-medium pb-6 border-solid border-b-4 border-blue-950 bg-opacity-75`,
```
Comment (adongare) [inline src/components/createTestCase/CreateTestCase.tsx]: looks like some classes are redundant(already there in line 142): h-2, font-medium.

border solid may not be needed. you have border 950.
does the opacity need?

File: src/components/createTestCase/CreateTestCase.tsx [hunk 2]
```diff
@@ -132,6 +133,17 @@ const INITIAL_VALUES = {
   groupPopulations: [],
 } as TestCase;
 
+interface PropTypes {
+  isActive?: boolean;
+}
+
+const MenuItemContainer = tw.ul`bg-transparent flex px-8`;
+const MenuItem = styled.li((props: PropTypes) => [
+  tw`relative h-2 mr-3 ml-3 mt-3 mb-0 not-italic font-rubik font-medium text-sm leading-4 text-gray`,
```
Comment (adongare) [inline src/components/createTestCase/CreateTestCase.tsx]: does this `text-grey` value the same as the one in design mockup #515151?

File: src/components/createTestCase/CreateTestCase.tsx [hunk 3]
```diff
@@ -566,151 +579,197 @@ const CreateTestCase = () => {
...
+            onChange={(val: string) => setEditorVal(val)}
+            value={editorVal}
+            setEditor={setEditor}
+            readOnly={!canEdit}
+          />
+        </div>
+
+        <div style={{ width: "17px", background: "#DDDDDD" }}></div>
+        <div tw="flex-auto sm:w-full md:w-1/12 lg:w-3/12">
+          <MenuItemContainer>
```
Comment (adongare) [inline src/components/createTestCase/CreateTestCase.tsx]: can the menu be its own component?

File: src/components/createTestCase/CreateTestCase.tsx (hunk 1 above)
Comment (sb-prateekkeerthi) [reply src/components/createTestCase/CreateTestCase.tsx]: Height is required property for maintaining the size of tabs column(it is also mentioned in the styling provided in the figma). Yeah I'll remove the remaining classes

File: src/components/createTestCase/CreateTestCase.tsx (hunk 2 above)
Comment (sb-prateekkeerthi) [reply src/components/createTestCase/CreateTestCase.tsx]: Thank you, I just changed it

File: src/components/createTestCase/CreateTestCase.tsx (hunk 3 above)
Comment (sb-prateekkeerthi) [reply src/components/createTestCase/CreateTestCase.tsx]: Thank you, I just moved it

**Review by adongare — APPROVED** [review]
looks good

**Pull request merged**